_config.yml

```diff
@@ -39,23 +39,23 @@ navigation:
   url: ato/
   internal: true
   children:
-  - text: Background
-    url: background/
```

Comment:
I'm actually working towards getting rid of the Background page all together. It is almost entirely duplicative with what is on the top level page.